contracts/contracts/vault/OETHBaseVaultAdmin.sol

```diff
@@ -8,5 +8,39 @@ import { VaultAdmin } from "./VaultAdmin.sol";
  * @author Origin Protocol Inc
  */
 contract OETHBaseVaultAdmin is VaultAdmin {
+    /**
+     * @notice Adds a strategy to the mint whitelist.
+     *          Reverts if strategy isn't approved on Vault.
+     * @param strategyAddr Strategy address
+     */
+    function addStrategyToMintWhitelist(address strategyAddr)
+        external
+        onlyGovernor
+    {
+        require(strategies[strategyAddr].isSupported, "Strategy not approved");
 
+        require(!mintWhitelistedStrategy[strategyAddr], "Already whitelisted");
+
+        mintWhitelistedStrategy[strategyAddr] = true;
+
+        emit StrategyAddedToMintWhitelist(strategistAddr);
+    }
+
+    /**
+     * @notice Removes a strategy from the mint whitelist.
+     * @param strategyAddr Strategy address
+     */
+    function removeStrategyFromMintWhitelist(address strategyAddr)
```

Comment:
since we don't check if strategy is valid here. Might be cool to check for a non zero address

Reply:
I guess that might be unnecessary. There should be no way to add a `address(0)` to the whitelist. `addStrategyToMintWhitelist` will fail when using `address(0)` since it isn't an approved strategy on Vault

Reply:
good point